### DISCUSSION TITLE
How do report Phishing/Spam pages?

### DISCUSSION THREAD

**Discussion by genman** [community/community#70334] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Select Topic Area

Question
How to report this page? we-apple-customer-reach-us.github.io

This link does not include anything relevant to this site: https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam

The options (which aren't applicable)
[Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
[Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
[Reporting a discussion](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-discussion)
[Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)
[Reporting an app in GitHub Marketplace](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-app-in-github-marketplace)
[Reporting contact link abuse in the template chooser](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-contact-link-abuse-in-the-template-chooser)

Fake site:
https://we-apple-customer-reach-us.github.io/care/?utm_medium=marketing&_branch_referrer=H4sIAAAAAAAAAxXKUQ9CQADA8U%2BjR6w0arN2DjFqqofKS0PidDh3p%2Bv69LH9n%2F771ZwTttU061W3hpoRomLUvbWDLG8Mp%2FTu5ruRYruembICytKfYiMhPeUzx6Va9O30FlWB0dOGjR7BUyOMHBGnhhKcQwZ8hkr4G4fAN0VjJkiaa7oy3GbfkkDPclJdNjREJRadx5NvFA%2BV%2FKRXVx6tDHDP8kAsHqJw%2FjGCNCusAAAA&url=https%3A%2F%2Fsupport.apple.com%2F&gclid=Cj0KCQjw4bipBhCyARIsAFsieCzuqHF7wj7Piy75r34DjGmpH0abpgS9rIielwnEtPxKLqgyvZWDyN8aAtE8EALw_wcB&_branch_match_id=1205936641237082101

Link leads you to a tech support scammer number: +1-(844)-781-6134

From:
https://www.reddit.com/r/ScamNumbers/comments/17a29nl/apple_support_scam_works_as_of_101723_1225pm_est/

How is there no easy way to report this?

**Comment by genman** [comment]
The previous "discussion" did not provide any specific information on this issue. If you cannot provide specific instructions, please do not close this matter.

**Comment by ghostinhershell** [comment]
Hey there @genman, thanks so much for reaching out about this. I get a 404 when trying to view the webpage. Are you still seeing the web page on your end?

**Comment by Ankman1** [comment]
Same problem here. I am not even sure if github.io is property of Github.

Otherwise a scammer sent me tiktokads . github.io (please remove spaces) which later wants to to fill out fake surveys from Amazon, Apple and the like.

If Github admins read here, please have a look at the URL and take it down.

**Comment by bene-ngrave** [comment]
I would also like to report a phishing site on Github Pages. This seems to be the only place where this can be reported. 
Or is there any other way that I should report this type of abuse, @ghostinhershell ? 
 
The domain is panelactivator(dot)github(dot)io/panel.activator/ They use this site to steal private keys of crypto wallets and they are actively promoting this url in a fake Telegram impersonating a crypto wallet provider.

**Comment by udanieli** [comment]
I want to report this phishing website too:
https://nellybata.github.io/
They send a link with a querystring parameter to trick any mail domain

**Comment by artmg** [comment]
There is no way to report the GitHub pages directly, however the gothub.io subdomain is the account name. If you would like to report an issue against, for example, `octocat.github.io` then the account name is **octocat** so point your browser at https://github.com/octocat and use the _Block or Report_ link under the user's profile info. 

Remember, don't actually use `octocat`, but substitute the subdomain under github.io that concerns you.

**Comment by ghostinhershell** [comment]
The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Reply by LambsFatAss** [reply]
How useless.

**Comment by joachimmetz** [comment]
How to report a project/organization that violates ToS?

**Comment by vc1cv1** [comment]
to report github.io pages -- do the following (using this as a phishing link: https://sarsoutstanding.github[.]io/atten/govoverdue.html)

- Probably needs you to be logged in
- Go to -> https://github.com/sarsoutstanding
- click 'report' right below he user's profile and the system should bring you to then create a case.
